README.md

```diff
@@ -1,5 +1,9 @@
 # HATiD
 
+<p align="center">
+<img alt="HATiD Logo" src="src/lib/assets/logo-type.svg" width="50%" />
```

Comment:
Just a nitpick: does Prettier flag when we add intentations?
```suggestion
    <img alt="HATiD Logo" src="src/lib/assets/logo-type.svg" width="50%" />
```

Reply:
It did not 😃